### ISSUE TITLE
Using app at same time

### ISSUE THREAD

**Issue by Ungov77** [NickWaterton/Roomba980-Python#100]
`only one connection at at time is allowed to the Roomba local mqtt server`

Does this mean I can't use the official iRobot app anymore if I use Roomba980 script?

**Comment by NickWaterton** [comment]
No, the iRobot app uses the cloud connection if the local connection is in use.

It will work normally.
